tfhub_dev/assets/rishit-dagli/chest-xray-covid/1.md

```diff
@@ -0,0 +1,9 @@
+# Placeholder rishit-dagli/chest-xray-covid/1
+Lightweight deep learning model for clasifying chest xrays as COVID or normal.
+
+<!-- module-type: image-classification -->
+<!-- network-architecture: other -->
```

Comment:
Is it possible to have a more specific description of the network-architecture? Same for dataset.

Reply:
I use a Neural Architecture Search based method to build the model and thus was not sure if I should fill out the architecture.

Reply:
I have added the dataset field.

Reply:
@maringeo Does this look good or would I need to enter the architecture details?